pykeyvi/requirements.txt

```diff
@@ -0,0 +1,5 @@
+msgpack-python
+autowrap
```

Comment:
I wonder if we should pin autowrap to a version, as this introduces a build time dependency to a API that does not look official to me.